fe/src/features/students/StudentPage.vue

```diff
@@ -12,188 +12,228 @@
-    </div>
     <div
-      v-if="authenticationStore.loggedInUser?.isAdmin ?? false"
-      class="row q-gutter-sm items-center"
+      :key="studentKey"
+      class="justify-between row q-px-lg q-pt-lg studentcol"
     >
-      <q-select
-        v-model="possibleFinalDecision"
-        emit-value
-        map-options
-        rounded
-        outlined
-        dense
-        style="width: 200px"
-        :options="[
-          { value: '0', label: 'Yes' },
-          { value: '2', label: 'Maybe' },
-          { value: '1', label: 'No' },
-          { value: '-1', label: 'Not decided' },
-        ]"
-        label="Final decision"
+      <div class="row q-px-sm q-gutter-sm items-center">
+        <h class="text-bold text-h4">
+          {{ student ? student.fullName : '' }}
+        </h>
+        <q-btn
+          :href="student ? student.cv.toString() : ''"
+          target="_blank"
+          size="12px"
+          rounded
+          outline
+          color="black"
+          label="CV"
+        />
+        <q-btn
+          :href="student ? student.portfolio.toString() : ''"
+          target="_blank"
+          size="12px"
+          rounded
+          outline
+          color="black"
+          label="Portfolio"
+        />
+      </div>
+      <div
+        v-if="authenticationStore.loggedInUser?.isAdmin ?? false"
+        class="row q-gutter-sm items-center"
+      >
+        <q-select
+          v-model="possibleFinalDecision"
+          emit-value
+          map-options
+          rounded
+          outlined
+          dense
+          style="width: 200px"
+          :options="[
+            { value: '0', label: 'Yes' },
+            { value: '2', label: 'Maybe' },
+            { value: '1', label: 'No' },
+            { value: '-1', label: 'Not decided' },
+          ]"
+          label="Final decision"
+        />
+        <q-btn
+          class="cornered"
+          outline
+          label="Confirm"
+          @click="finalDecision"
+        />
+      </div>
+    </div>
+    <div class="row q-px-lg q-ml-sm q-mt-sm items-center">
+      <InfoDiv
+        v-if="student?.alum"
+        use-icon="mdi-account-school"
+        color="blue"
+        title="Alumni"
+      />
+      <InfoDiv
+        v-if="student?.studentCoach"
+        use-icon="mdi-account-group"
+        color="yellow"
+        title="Student coach"
+      />
+      <InfoDiv
+        v-if="student?.employmentAgreement"
+        use-icon="mdi-file-document-edit"
+        color="red"
+        content="Employment"
+        :title="employment"
+      />
+      <InfoDiv
+        v-if="student?.gender"
+        :use-icon="genderIcon"
+        color="green"
+        content="Gender"
+        :title="gender"
       />
+      <q-space />
       <q-btn
         class="cornered"
         outline
-        label="Confirm"
-        @click="finalDecision"
+        color="red"
+        icon-right="delete"
+        label="Delete"
+        @click="showDeleteDialog"
```

Comment:
You can just do `deleteDialog = true` here since this function does nothing else